ci/terraform-lint.tests.ps1

```diff
@@ -0,0 +1,32 @@
+
+$RootPath = Split-Path -Parent $MyInvocation.MyCommand.Path | Split-Path
```

Comment:
Kara can you give me explain what this code is trying to achieve.

Reply:
https://ss64.com/ps/syntax-myinvocation.html